src/platform_impl/linux/wayland/event_loop/mod.rs

```diff
@@ -601,6 +617,19 @@ impl<T: 'static> EventLoop<T> {
     }
 }
 
+#[derive(Clone)]
+pub struct OwnedDisplayHandle {
+    display: Display,
+}
+
+impl OwnedDisplayHandle {
+    pub fn raw_display_handle(&self) -> RawDisplayHandle {
+        let mut display_handle = WaylandDisplayHandle::empty();
+        display_handle.display = self.display.get_display_ptr() as *mut _;
+        RawDisplayHandle::Wayland(display_handle)
```

Comment:
you can now do `into()` instead, though, I'd suggest to update all such places to call into given that you're updating the raw-window-handle crate.